web/taskmaster/src/app/services/chat.service.ts

```diff
@@ -7,6 +7,7 @@ import { environment } from '../../environments/environment';
 })
 export class ChatService {
   private socket: Subject<MessageEvent>;
+  public messageReceived$ = new Subject<boolean>();
```

Comment:
You can use a void subject Subject<void>() instead since you don't care about the boolean value. Then call it with `this.chatService.messageReceived$.next();` in the chat component.